sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -357,7 +357,6 @@ object RapidsConf {
         "back into GPU memory temporarily. Unspilling may be useful for GPU buffers that are " +
         "needed frequently, for example, broadcast variables; however, it may also increase GPU " +
         "memory usage")
-      .internal()
       .booleanConf
       .createWithDefault(false)
```

Comment:
Do we still want to default this to `false`?  Do we have any cases where it is known to hurt performance?

Reply:
With GDS spilling it seems like a net win, but with host memory spilling it's less clear, maybe slightly slower.